vips/vip-110.ts

```diff
@@ -0,0 +1,340 @@
+import { ProposalType } from "../src/types";
```

Comment:
I would organize all files related to VIP-110 under dir `vip-110` so this becomes `vips/vip-110/index.ts`

Reply:
...
`vips/vip-110/testnet-addendum3.ts`
`vips/vip-110/testnet.ts`
...
etc